guides/common/modules/proc_using-insights-with-satellite-server.adoc

```diff
@@ -14,7 +14,7 @@ To maintain your {ProjectServer}, and improve your ability to monitor and diagno
 .Scheduling insights-client
 
 Note that you can change the default schedule for running `insights-client` by configuring `insights-client.timer` on {Project}.
-For more information, see https://access.redhat.com/documentation/en-us/red_hat_insights/2020-10/html/client_configuration_guide_for_red_hat_insights/changing-the-client-schedule[Changing the insights-client schedule] in the _Client Configuration Guide for Red Hat Insights_.
+For more information, see {RHDocsBaseURL}red_hat_insights/2020-10/html/client_configuration_guide_for_red_hat_insights/changing-the-client-schedule[Changing the insights-client schedule] in the _Client Configuration Guide for Red Hat Insights_.
```

Comment:
This link gives a 404. I'm not familiar with Insights docs but I think this is the up-to-date one:
```suggestion
For more information, see {RHDocsBaseURL}red_hat_insights/1-latest/html/client_configuration_guide_for_red_hat_insights/assembly-client-changing-schedule[Changing the insights-client schedule] in the _Client Configuration Guide for Red Hat Insights_.
```

Reply:
The exact file rename looks like a regression while the version makes sense. Is that something we should let the insights team know?

Reply:
Ok, I pinged the Insights docs team in an internal chat and shared a link to this comment thread. I don't think it should block this PR; if there is something to change on the Insights docs' side, they know where to start investigating.

Reply:
The Insights docs team has confirmed that the suggested link is correct and follows their file naming guidelines: https://docs.redhat.com/en/documentation/red_hat_insights/1-latest/html-single/client_configuration_guide_for_red_hat_insights/index#assembly-client-changing-schedule